Scout-EUSS.py

```diff
@@ -0,0 +1,102 @@
+from buildstock_query import BuildStockQuery
+import pandas as pd
+import numpy as np
+
+upgrade_num = 3 #define upgrade number, change this number for different upgrade packages
+
+# get county name list for each EMM region
+emm_mapping = pd.read_csv('county_to_emm.csv')
+emm_county = {k: g["county"].tolist() for k,g in emm_mapping.groupby("emm")}
+
+my_run = BuildStockQuery(db_name='euss-final',
+                        table_name='euss_res_final_2018_550k_20220901',
+                        workgroup='scout',
+                        buildstock_type='resstock')
+
+def end_use(building_temp, scout_col2resstock, agg_functions):
+    building = building_temp.groupby(building_temp['time']).aggregate(agg_functions)
+    #add hour of year
+    building.insert(loc=0, column='Hour of Year', value=np.arange(len(building), dtype=int)+1)
+    # delete the end use columns that doesn't have savings
+    for resstock_col_prefixes in scout_col2resstock.keys():
+        if (building[f"{resstock_col_prefixes}__savings"] >= 0).all(axis=0):
+            building = building.drop([f"{resstock_col_prefixes}__baseline"], axis=1)
+            building = building.drop([f"{resstock_col_prefixes}__savings"], axis=1)
+            
+    # save individual end use data to results
+    building_results = pd.DataFrame(columns=['Hour of Year', 'EMM Region', 'Building Type'])
+    num_end_use = int((building.shape[1]-3)/2)
+    for j in range (num_end_use):
+        building_end_use = building[['Hour of Year','EMM Region', 'Building Type']]
+        building_end_use['End Use'] = building.columns[4+j*2].split('__')[0]
+        building_end_use['Baseline'] = building[building.columns[4+j*2]]
+        building_end_use['Measure'] = building[building.columns[4+j*2]]-building[building.columns[5+j*2]]
+        frames = [building_results, building_end_use]
+        building_results = pd.concat(frames)
+    return(building_results)
```

Comment:
```suggestion
    return building_results
```
We don't need brackets in return statement.